### DISCUSSION TITLE
Adding color swatches to the Color Fieldtype

### DISCUSSION THREAD

**Discussion by mefenlon** [statamic/cms#9468] [Q&A] [answered]
I am  new to Tailwind and attempting to define new custom color names and overwrite a few color values for use in the Color Fieldtype.

The color swatch values that are predefined in tailwind display as their predefined values (blue, indigo...), and I am unable to display any of the colors I am attempting to generate.

I have followed the tailwind instructions of adding my color names as described here: https://tailwindcss.com/docs/customizing-colors#adding-additional-colors
I have created a tailwind tailwind.config.js file in my project directory.
```
/** @type {import('tailwindcss').Config} */
module.exports = {
    theme: {
        extend: {
            colors: {
                brown: '#4f371b',
                'blue-alt': '#98b4dc',
            }
        },
    },
}
```

I am running into issues getting these new values to appear on the cp.
I have followed this: https://statamic.dev/extending/control-panel#adding-assets-to-your-build-process
And am not getting the desired result
<img width="287" alt="Screenshot 2024-02-06 at 3 44 56 PM" src="https://github.com/statamic/cms/assets/30707325/87fe15cb-0940-4c55-9032-8f01343920f5">

**Comment by jasonvarga** [comment, marked as answer]
The color field doesn't integrate with your tailwind configuration. It just accepts hex codes.

You may consider this addon instead: https://statamic.com/addons/krakero/tailwind-fieldtype
